nextflow.config

```diff
@@ -149,6 +149,28 @@ params {
     damageprofiler_length        = 100
     damageprofiler_threshold     = 15
     damageprofiler_yaxis         = 0.3
+
+    // Damage Manipulation options
+    run_mapdamage_rescaling                                          = false
+    damage_manipulation_rescale_seqlength                            = 12
+    damage_manipulation_rescale_length_5p                            = 0
+    damage_manipulation_rescale_length_3p                            = 0
+    run_pmd_filtering                                                = false
+    damage_manipulation_pmdtools_threshold                           = 3
+    damage_manipulation_pmdtools_reference_mask                      = null
```

Comment:
MIssing `--pmdtools_range`, `--pmdtools_max_reads`, `--pmdtools_playtpus` from eager2

Reply:
Can you make an issue about that, I didn't realise we run PMDtools twice

Reply:
We didn't use to, but we have now separated the damage manipulation and estimation